README.md

```diff
@@ -5,31 +5,45 @@ PyPI.
 
 
 ## Usage
-To use the action simply add the following lines in the end of your
-`.github/main.workflow`.
-
-```hcl
-action "Upload Python dist to PyPI" {
-  uses = "re-actors/pypi-action@master"
-  env = {
-    TWINE_USERNAME = "f'{your_project}-bot'"
-  }
-  secrets = ["TWINE_PASSWORD"]
-}
+
+To use the action add the following step to your workflow file (e.g.:
+`.github/workflows/main.yml`)
+
+
+```yml
+- name: Publish package
+  uses: pypi/gh-action-pypi-publish@master
+  with:
+    user: __token__
+    password: ${{ secrets.pypi }}
+```
+
+A common use case is to upload packages only on a tagged commit, to do so add a
+filter to the step:
+
+
+```yml
+  if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
 ```
 
-N.B. Use a valid tag, or branch, or commit SHA instead
-of `master` to pin the action to use a specific version of it.
+So the full step would look like:
+
+
+```yml
+- name: Publish package
+  uses: pypi/gh-action-pypi-publish@master
+  if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
```

Comment:
I'd put this one line above. `uses` and `with` should go together because they share a common context.